src/routes/console/organization-[organization]/usage/[[invoice]]/+page.ts

```diff
@@ -58,6 +58,7 @@ export const load: PageLoad = async ({ params, parent }) => {
         organizationUsage: usage,
         projectNames: projectNames.projects,
         invoices,
-        currentInvoice
+        currentInvoice,
+        organizationMembers: await sdk.forConsole.teams.listMemberships(params.organization)
```

Comment:
we can fetch those with one of the others 👍🏻 no need for this to be blocking

Reply:
What do you mean by one of the others? :sweat_smile: